Systems biology approaches for understanding SARS-CoV-2 pathogenesis (#68)
greenelab/covid19-review: content/07.pathogenesis.md

### Systems level approaches for understanding SARS-CoV-2 pathogenesis

> Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@10.1038/s41576-020-0212-5]. Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@10.1002/prca.201300074].

Reviewer: ```suggestion
Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
```

Please use a newline "Enter key" after each sentence.  That makes it easier for experts to give comments on each sentence and for us to track revisions.  This paragraph would be converted to what I have above.

The references look better but need to have the `doi:` prefix.  I added them in this paragraph as an example.

### Systems level approaches for understanding SARS-CoV-2 pathogenesis

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].

> Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].

Reviewer 2: ```suggestion
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
```

Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].

The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 

> A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 

Reviewer 2: Two suggestions beyond just some minor flow suggestions:
1. I believe SARS-CoV-2 is the more up-to-date name (per https://www.nature.com/articles/s41564-020-0695-z)
2. These citations are causing a bug, but I think it's just because DOIs that have "(" or ")" sadly cause issues. For future reference, you can convert them to remove these characters at this link: http://shortdoi.org/. I believe it should work with the change below.
```suggestion
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7]. 
```

> Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 

Author: Corrections seem good

> Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 

Author: Referring to the individual sample types separately seems more precise

> Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10.1016/S0140-6736(20)30251-8]. 

Reviewer 2: Same DOI as above
```suggestion
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43]. 
```

> However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7].

Reviewer 2: ```suggestion
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
```

> The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 

Author: It refers to the 3 genome sequence papers - One Lancet and Two Natures

> The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 

Author: I have removed the immunology related discussion from this sub-section to avoid redundancy. We will add those and other suggested papers in the "Immune Response to SARS-CoV-2" sub-section.

> The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 

Reviewer 3: Briefly corresponded with @rando2 . This was good to go for now & we will continue to revise as more is learned 👍

> In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 

Reviewer 4: There was an interesting paper published yesterday showing the induction of neutralizing antibodies against the viral spike protein receptor-binding domain in humans post SARS-COV-2 infection: https://www.biorxiv.org/content/10.1101/2020.03.21.990770v1
An interesting paper that certainly merits discussion when thinking about epitope-specific responses. I can spend some time trying to identify similarities between the predicted epitopes and observed antibodies.

Reviewer 5: To add to this, nice report on the breadth and kinetics of immune cells mobilized in response to SARS-CoV2 infection, suggesting a concerted T cell and B cell response is needed for effective viral clearance: https://www.nature.com/articles/s41591-020-0819-2

Reviewer 6: @rishirajgoel: I am currently writing on a neutralizing antibody section in the treatments section. You are more then welcome to participate in that!

> In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 

Reviewer 2: ```suggestion
In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
```

> Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10/ggk6c5].

Reviewer 4: The Lancet Respiratory Med paper certainly shows hyperactivation of CD4+ and CD8+ cells (incl high expression of cytotoxic granule proteins in the CD8+s), but their overall count was reduced. The authors specifically note an expansion of the Th17 expansion among CD4+ cells and speculate that these cells may promote tissue damage. There is some evidence to support these findings in other viral infections (I can provide citations) and so this is an interesting hypothesis).

Semantic note: the overactivation of T cells is not "caused" by an increase in Th17 and cytotoxic CD8+s. Rather, these findings indicate an activated T cell phenotype. 

Another group published a study looking at T cell phenotype here: https://www.nature.com/articles/s41423-020-0401-3

My summary of the major findings --> increased levels of HLA-DR (MHC-II) and TIGIT in CD8+s. No differences in activation markers on CD4+. Interestingly, HLA-DR/TIGIT on CD8+ were observed at higher levels in the severe vs mild COVID group, providing some more evidence that CD8+ activation and may be important in pathology. To me it's a bit unclear what is happening w/ the CD4+ cells...

Reviewer 4: I should add that the Lancet Resp Med paper only phenotyped 1 patient and therefore should be considered with some caution. The cellular and molecular immuno paper I linked followed a larger cohort and reported similar findings in the CD8+ compartment so I think that's probably a solid angle to pursue.

Author: Yes, the Xu et al. Lancet Respir Med. 2020 is a case report from Wuhan, China (Treatment and Research Center for Infectious Diseases). Considering a single patient report (which is the same for many other top-tier publications on 2019-nCoV at this stage) the phenomenon is stated as "could be". The findings from this paper are highly interesting. Anyway, please feel free to include needful modifications.

Author: Many thanks for your valuable input.

Reviewer 4: Agreed! And my pleasure. Lots of interesting things to think about. If you think the Cellular/Molecular Immuno paper merits inclusion in this section I can add a brief comment in the main text.

Reviewer 5: Great discussion. I would add a few other potential references of interest including:

https://www.medrxiv.org/content/10.1101/2020.02.23.20026690v1
- single-cell seq of BAL also backs a dysregulated CD8 angle
- limited sample-size, but interesting to note potential sources of local cytokine storm in lungs
- caveat = all patients had pneumonia (including mild ones) which was surprising to me

https://www.medrxiv.org/content/10.1101/2020.02.18.20024364v1
- shows clear drops in T cell numbers
- implicates PD-1 and Tim3 in exhaustion as well
- inverse correlation between some circulating cytokines and T cell numbers

https://academic.oup.com/nsr/advance-article/doi/10.1093/nsr/nwaa041/5804736
- implicates Th1 subset in severe disease
- raises role for monocyte subpopulation as well

https://www.medrxiv.org/content/10.1101/2020.02.16.20023903v1
- decreased IFNg+ CD4+ T cells in severe patients
- further evidence of proteins involved in cytokine storm in severe COVID19

https://www.nature.com/articles/s41423-020-0402-2
- evidence of increased exhaustion of cytotoxic cells (CD8+ T cells and NK cells) in severe patients
- analysis of NKG2A exhaustion 

https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa248/5803306
- another subset analysis looking at CD4 and CD8 naive, helper, CD4 Tregs
- higher naive, and lower memory and Tregs in severe cases

Reviewer 5: I would also encourage conservative interpretation of the Th17 finding in Lancet Resp Med. They show CCR6+ Th17s but don't use IL-17A staining to prove they are bonafide and functional Th17s. As shown in many other references, CCR6+ populations of CD4+ T cells can have functional diversity as noted in other viral infections: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995622/

Reviewer 3: This is really helpful. There's a lot of good stuff here. I'd propose this. I'm going to try to suggest a revision to be more conservative about Th17. Then, perhaps, we can get @rishirajgoel to come back through and suggest edits/revision to incorporate the additional papers mentioned.

At that point, perhaps @gregszetoAI and add the breadth of the literature that he raised. How does that sound?

Author: The statement on the findings from Xu et al. Lancet Respir Med. 2020 is tuned down as suggested. 
I think it might be a good idea to include further details in the "Immune Response to SARS-CoV-2" subsection under in the Pathogenesis section. 
Discussions on the papers suggested by @gregszetoAI and @rishirajgoel will be most suitable for that sub-section. What do you think @cgreene and HM Rando@rando2?

Reviewer 5: Mostly agree. Given the systems bio section, I would think at least the single-cell seq paper may fit well here.

Reviewer 6: I agree with @rays1987. I think the pathogenesis section needs some experts in virology and immunology and it seems like you guys have a good background in that. I outlined the pathogenesis section relatively early on just to spark some idea so feel free to adjust it to your needs but I think that understanding the basics of SARS-CoV-2 entry, replication and the immune response is obviously one the heart of COVID-19 biology. My suggestion would be that the systems part of this section should be more focused on the technologies and less on the biology. Most papers take advantage of systems approaches these days so it might also be worth thinking about splitting the systems bio part into where it fits into biology to avoid redundancy.

> Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10.1016/S2213-2600(20)30076-X].

Reviewer 2: ```suggestion
Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10/ggk6c5].
```

> Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. 

Reviewer 2: ```suggestion
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis. 
```

> Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently [@doi:10.1126/science.abb2507]. 

Reviewer 2: ```suggestion
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently [@doi:10.1126/science.abb2507]. 
```

> Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. 

Reviewer 2: ```suggestion
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. 
```

The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.

> There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 

Reviewer 2: ```suggestion
There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 
```

> Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074]. 

Reviewer 2: ```suggestion
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074]. 
```

> Considering the paucity of omics-level big data sets on for SARS-CoV-2 till now, the existing data hubs that contain adequate information for the other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), The Virus Pathogen Resource (ViPR) could be the useful resources for computational and bioinformatics research on SARS-CoV-2.

Reviewer 2: ```suggestion
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
```

### Systems level approaches for understanding SARS-CoV-2 pathogenesis

> [Sandipan Ray volunteered to lead this section]

Reviewer 3: ```suggestion
```

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. 

> These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases (Hood and Tian, 2012, PMID: 23084773). 

Reviewer: The next step will be adding in the Manubot-style automatic references.  For example, this manuscript has a DOI, which you can get from PubMed or the journal website
![image](https://user-images.githubusercontent.com/3532898/77592593-7c83a580-6ec0-11ea-8870-d2fe1e773a90.png)

The DOI is the best identifier to use if the article has one.  You can replace the citation with the Manubot DOI citation syntax:
```suggestion
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004]. 
```

Now Manubot will automatically pull in the reference details!

Author: Many thanks for this suggestion. Please could you suggest how do it when there are more than one references? Are those need to be separated by , or ;

Author: I have resolved the issue. Please have a look.

> Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases (Eckhardt et al., 2020, PMID: 32060427). 

Reviewer 3: ```suggestion
Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5]. 
```

Author: @cgreene
Citation issue resolved
Please review "Systems biology approaches for understanding SARS-CoV-2 pathogenesis  #68"

Reviewer 3: I see that the one that @agitter suggested was resolved and I suggested another one. There are a lot of references that don't appear to use the manubot style though. Can you take a screenshot of what you're seeing as you're resolving them?

Author: Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@10.1038/s41576-020-0212-5]. Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@10.1002/prca.201300074].
The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@10.1038/s41586-020-2008-3; @10.1016/S0140-6736(20)30251-8; @10.1038/s41586-020-2012-7]. Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@10.1016/S0140-6736(20)30251-8]. However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@10.1016/S0140-6736(20)30251-8; @10.1038/s41586-020-2012-7].
The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@10.1016/j.chom.2020.03.002]. Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@10.1016/S2213-2600(20)30076-X].
Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently [@10.1126/science.abb2507]. Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@10.1016/j.cell.2020.02.052]. Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@10.1073/pnas.0407992101; @10.1016/j.bbrc.2004.02.074]. Considering the paucity of omics-level big data sets on for SARS-CoV-2 till now, the existing data hubs that contain adequate information for the other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), The Virus Pathogen Resource (ViPR) could be the useful resources for computational and bioinformatics research on SARS-CoV-2.
